index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Twitter Page</title>
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
+    <link rel="stylesheet" href="master.css">
+    <link href="./images/favicon.ico" rel="icon" type="image/x-icon" />
+</head>
+<body>
+    <section class="whiteBlue">
+        <section id="blue">
+            <div id="blueContent">
+                <li class="icons"> <span class="iconify" data-icon="entypo-magnifying-glass" data-inline="false"></span>Follow your Interest.</li>
+                <li  class="icons"> <span class="iconify" data-icon="ion:people" data-inline="false"></span>Hear what people are talking about. </li>
+                <li class="icons"> <span class="iconify" data-icon="emojione-v1:right-speech-bubble" data-inline="false"></span>Join the conversation.</li>       
+            </div>
+            
+        </section>
+            <section id="white">
+                <div id="topWhiteContent">
+                    <input class="input" type="text" id="phone">
+                    <input class="input" type="text" id="password">
+                    <a href="">forgot password ?</a>
+                    <button id="upperlogin">Log in</button>
+                </div>
+                <div id="lowWhiteContent">
+                    <h1>See what's happening in the world right now</h1>
+                    <h2>Join Twitter today.</h2>
+                        <button id="signup">Sign Up</button>
+                        <button id="Login">Log in</button>
+                </div>
+                </section>
+    </section>
+    <section id="footer">
+        <footer>
+            <div>
```

Comment:
this div is a little unnecessary. Just like the `<section id="footer">` div. 
Your `<ul>` tag is already a div within your `<footer>` div. That is already enough to work with.